packages/chat/src/views/View.js

```diff
@@ -112,10 +112,18 @@ const Textarea = ({ state, dispatch }) => {
   return (
     <textarea
       className='w-full' rows='5' value={value(state, 'content')} onChange={change.bind(null, dispatch, 'content')}
+      onKeyDown={e=>{handleMetaEnter(e)}}
     />
   )
 }
 
+const handleMetaEnter = e => {
+  if(e.key === 'Enter' && (e.metaKey || e.ctrlKey)){
+    e.preventDefault()
+    e.target.form.requestSubmit()
```

Comment:
Thanks for suggesting this feature!

AFAIK, `requestSubmit` is not supported by Safari neither on desktop nor on iOS.

We should find an alternative and maybe either call `submit` on the form or then just call the `sendMessage` fn directly.

Open to other suggestions as long as it works across all major browsers on desktop and mobile.